etc/calico/confd/templates/bird_ipam.cfg.template

```diff
@@ -1,6 +1,16 @@
 # Generated by confd
 filter calico_pools {
   calico_aggr();
+{{- $static_key := "/staticroutes"}}
```

Comment:
Might want to consider renaming this filter eventually, since it's not really ip pool specific. 

Not a big deal though.

Reply:
I will do that in the follow-on template clean-up PR.